x-pack/legacy/plugins/fleet/server/libs/__mocks__/token.ts

```diff
@@ -15,7 +15,7 @@ export class TokenLib {
         return {
           valid: true,
           type: TokenType.ENROLMENT_TOKEN,
-          token: { config: { id: 'configId', sharedId: 'configSharedId' } },
+          token: { config: { id: 'policyId', sharedId: 'configSharedId' } },
```

Comment:
Again config should be renamed in the key name

Reply:
Yes missed this one